src/main/java/de/tum/in/www1/artemis/service/metis/conversation/errors/ChannelNameDuplicateException.java

```diff
@@ -24,7 +24,7 @@ private static Map<String, Object> getParameters(String channelName) {
         Map<String, String> params = new HashMap<>();
         params.put("channelName", channelName);
         Map<String, Object> parameters = new HashMap<>();
-        parameters.put("skipAlert", true);
+        parameters.put("skipAlert", false);
```

Comment:
Is false the default value here? If so, could you leave out this line?

Reply:
By default the value is not set. I removed the line setting parameter to false.